dashboard/config/linux/bits/base.yml

```diff
@@ -83,7 +83,7 @@ config:
  - PANIC_ON_OOPS
  - PANIC_TIMEOUT: 86400
  - SCHED_STACK_END_CHECK
- - FORTIFY_SOURCE: [-riscv, -clang]
+ - FORTIFY_SOURCE: [-riscv, v5.18]
```

Comment:
If a user configures a 5.17 kernel with GCC, my understanding is that this will "regress" from using FORTIFY_SOURCE to not using it. See the `dashboard/config/linux/stable-5.4-kasan.config` diff for example. But I'm not sure if the syz-kconf syntax is able to express more complex expressions and if it's worth overthinking those.

Reply:
This configuration language does not support complex constraints with arbitrary and/or. So it will either need to be extended, or this somehow split into separate files for clang/gcc, or if we don't have any such kernels on syzbot, we can just go with this change.

Reply:
Yes, we'll get a regression for older kernels. But e.g. `dashboard/config/linux/stable-5.4-kasan.config` is not even used anymore (should we delete it btw?), so we should be fine.